docs/source/AdministratorGuide/Systems/WorkloadManagement/Pilots/index.rst

```diff
@@ -131,6 +131,14 @@ The pilot can be configured to run in several ways.
 Please, refer to https://github.com/DIRACGrid/Pilot/blob/master/Pilot/pilotTools.py
 for the full list.
 
+Pilot status
+============
+
+The pilot status of a successful pilot proceeds in the following order:
+- Submitted: the pilot goes from DIRAC to a remote site
```

Comment:
Needs a new line before the bullet list
https://dirac--5314.org.readthedocs.build/en/5314/AdministratorGuide/Systems/WorkloadManagement/Pilots/index.html?highlight=pilot%20status#pilot-status